src/index.js

```diff
@@ -101,6 +101,7 @@ const formatOptions = ({steps, ...otherOptions}) => ({
 
 
 Onfido.init = (opts) => {
+  console.log("onfido_sdk_version", process.env.SDK_VERSION)
```

Comment:
yes, because I wanted clients to be able to inform us of their sdk version. but maybe I can just put it in the `Onfido` object and call it `VERSION`. thoughts?